package.json

```diff
@@ -67,5 +70,6 @@
                 }
             }
         }
-    }
+    },
+    "packageManager": "pnpm@9.14.2+sha512.6e2baf77d06b9362294152c851c4f278ede37ab1eba3a55fda317a4a17b209f4dbb973fb250a77abc463a341fcb1f17f17cfa24091c4eb319cda0d9b84278387"
```

Comment:
Why is this added?

Reply:
I don't know why this happened, but this line was added in the beginning, when I ran the install 🤔 
Anyway, I removed the line and it wasn't generated anymore: 47ad173c8ebb0383a8df44eb73c949ef80e02188